src/doc/rustdoc/src/the-doc-attribute.md

```diff
@@ -0,0 +1,108 @@
+# The `#[doc]` attribute
+
+The `#[doc]` attribute lets you control various aspects of how `rustdoc` does
+its job. 
+
+The most basic job of `#[doc]` is to be the way that the text of the documentation
+is handled. That is, `///` is syntax sugar for `#[doc]`. This means that these two
```

Comment:
I find this phrasing (and the repetition of "job") really awkward... how about "The most basic function of `#[doc]` is to handle the actual documentation text."?